pingcap/tiup
# Distinct instanceDir and wd

### What problem does this PR solve? 
Fix the issue that `tiup_process_meta` is generated at the wd and `tiup status` can not work.

### What is changed and how it works?
After #682, we do not change wd for the run components.
It make it save the tiup_process_meta file in wd instead of
InstanceDataDir like before. This seems broke "tiup clean" and "tiup
status" which check all subdirectory in ~/.tiup/data for the
tiup_process_meta.

After this PR, we Distinct InstanceDataDir and wd in launchComponent and
make Dir in tiup_process_meta as the InstanceDataDir instead of wd.


### Check List 

Tests 


 - Manual test (add detailed scripts or steps below)
run `tiup tidb` and check `tiup status`




Release notes:

```release-note
NONE
```
